Wikipedia/Code/CollectionViewEditControllerNavigationDelegate+Extensions.swift

```diff
@@ -8,10 +8,10 @@ extension CollectionViewEditControllerNavigationDelegate where Self: UIViewContr
             batchEditToolbarViewController.view.translatesAutoresizingMaskIntoConstraints = false
             view.addSubview(batchEditToolbarViewController.view)
             batchEditToolbarViewController.didMove(toParentViewController: self)
-            
+            let height = max(view.safeAreaInsets.bottom, 60)
```

Comment:
should this be `view.safeAreaInsets.bottom + 44`? seems like the height should never == safeAreaInsets.bottom

Reply:
in `Saved`, if we did `view.safeAreaInsets.bottom + 44` it would be `128`, which seems pretty tall 🤔 `view.safeAreaInsets.bottom` is `84` in the `Saved` tab